elfpy/pricing_models/yieldspace.py

```diff
@@ -44,7 +45,7 @@ def calc_lp_out_given_tokens_in(
         rate: float,
         market_state: hyperdrive_market.MarketState,
         time_remaining: time.StretchedTime,
-    ) -> tuple[float, float, float]:
+    ) -> Tuple[float, float, float]:
```

Comment:
Why did you change this? I think the lower-case `tuple` is correct, and the upper-case `Tuple` is being deprecated.

Reply:
oops I thought I saw way more higher case Tuples in our code, i'll change it back